tests/test_runs/test_pipelines/configs/train_arcface_with_categories.yaml

```diff
@@ -51,17 +51,19 @@ metric_args:
   return_only_overall_category: True
   visualize_only_overall_category: True
 
-log_images: False
+log_images: True
 
 metric_for_checkpointing: OVERALL/cmc/1
 
 max_epochs: 2
 valid_period: 1
 
 logger:
-  name: tensorboard
+  name: clearml
```

Comment:
please, add the following string to the corresponding test:

```
@pytest.mark.long
@pytest.mark.needs_optional_dependency  # <---------- THIS ONE
@pytest.mark.parametrize("accelerator, devices", accelerator_devices_pairs())
def test_train_arcface_with_categories(accelerator: str, devices: int) -> None:
    run("train_arcface_with_categories.py", accelerator, devices)
```

Reply:
Done